Nodejs/Product/Nodejs/Project/NodejsProjectLauncher.cs

```diff
@@ -350,7 +350,11 @@ private void StartWithChromeV2Debugger(string program, string nodeRuntimeExecuta
 
             var setupConfiguration = new SetupConfiguration();
 
-            var visualStudioInstallationInstanceID = setupConfiguration.GetInstanceForCurrentProcess().GetInstanceId();
+            var setupInstance = setupConfiguration.GetInstanceForCurrentProcess();
+
+            var visualStudioInstallationInstanceID = setupInstance.GetInstanceId();
+
+            var pathToNodeExe = setupInstance.GetInstallationPath() + "\\JavaScript\\Node.JS\\v6.4.0_x86\\Node.exe";
```

Comment:
Can you switch this to `Path.Combine`, and add a comment why we need this specific version of Node? 

Is the following worth investigating: have the node installation package write a key to the registry, and have this component read that key... that way when the version of node is updated (or the path is changed), we don't have to remember to make changes here too.